assembl/static2/js/app/components/search.jsx

```diff
@@ -108,15 +108,22 @@ if (v1Interface) {
     case 'idea':
       ideaBase64id = btoa(`Idea:${id}`);
       return `/${slug}/debate/thread/theme/${ideaBase64id}`;
-    default:
+    default: {
       // post
       ideaId = hit._source.idea_id.length > 0 ? hit._source.idea_id[0] : null;
       if (!ideaId) {
         return undefined;
       }
       ideaBase64id = btoa(`Idea:${ideaId}`);
       postBase64id = btoa(`Post:${id}`);
-      return `/${slug}/debate/thread/theme/${ideaBase64id}/#${postBase64id}`;
+      const phaseId = hit._source.phase_id || 'thread';
+      if (phaseId === 'thread') {
+        return `/${slug}/debate/thread/theme/${ideaBase64id}/#${postBase64id}`;
```

Comment:
shouldn't you use routes defined in `routes.json`?

Reply:
yes, done

Reply:
Doubly agree with @cedricmessiant . You can use existing routes to form these URLs...